.github/workflows/elixir-build-check.yaml

```diff
@@ -58,3 +58,21 @@ jobs:
             _build/dev/*.plt
             _build/dev/*.plt.hash
       - run: mix do compile --force, dialyzer
+  dependency_audit:
```

Comment:
Nitpick: I have used dashes instead of underscores elsewhere in job and step id/name. I personally find that more readable, and it is [valid syntax](https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow#setting-an-id-for-a-job).

Reply:
PR updated.